maintenance-branch

```diff
@@ -1 +1 @@
-4-0-dev
+drop-old-rubies
```

Comment:
⚠️ This change is temporary and needs to be removed before merging, or soon after ⚠️

Reply:
This PR should be capable of running off 4-0-dev? What makes it dependant?

Reply:
Reverted 👍